common/flavorization/00_CTP.txt

```diff
@@ -6526,6 +6526,399 @@ duke_tribal_female_italian = {
 ## End of Latin_cultrue_group ##
 ###############################################
 
+
+###############################################
+## Eastern_slavic_cultrue_group ##
+###############################################
+
+baron_male_east_slavic_group = {
+	type = character
+	gender = male
+	special = holder
+	tier = barony
+	priority = 7
+	cultures = { east_slavic_group }
+	governments = { feudal_government tribal_government clan_government }
+	top_liege = no
+}
+baron_female_east_slavic_group = {
+	type = character
+	gender = male
+	special = holder
+	tier = barony
+	priority = 7
+	cultures = { east_slavic_group }
+	governments = { feudal_government tribal_government clan_government }
+	top_liege = no
+}
+baron_republic_male_east_slavic_group = {
+	type = character
+	gender = male
+	special = holder
+	tier = barony
+	priority = 7
+	cultures = { east_slavic_group }
+	governments = { republic_government }
+	only_holder = yes
+	top_liege = no
+}
+baron_republic_female_east_slavic_group = {
+	type = character
+	gender = female
+	special = holder
+	tier = barony
+	priority = 7
+	cultures = { east_slavic_group }
+	governments = { republic_government }
+	only_holder = yes
+	top_liege = no
+}
+barony_republic_east_slavic_group = {
+	type = title
+	tier = barony
+	priority = 7
+	cultures = { east_slavic_group }
+	governments = { republic_government }
+	top_liege = no
+}
+
+### Russian ###
+# Feudal
+emperor_male_russian = {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = 102
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { russian }
+}
+emperor_female_russian = {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = 102
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { russian }
+}
+empire_feudal_russian = {
+	type = title
+	tier = empire
+	priority = 102
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { russian }
+}
+title_prince_male_russian_empire = {
+	type = character
+	gender = male
+	special = ruler_child
+	tier = empire
+	priority = 40
```

Comment:
The title_prince__empire has a priority lesser than king_male_russian. This mean that the king title is usually preferred over the title prince__empire. If this is intended, then its okay

Reply:
This should be intended; it would be weird as hell to see "tsarevich" instead of the king title.